packages/components/src/Tile/subcomponents/GenericTile/GenericTile.module.scss

```diff
@@ -60,36 +62,40 @@ $tilePaddingTop: $spacing-xl;
 
 .faceFront {
   justify-content: space-between;
+  border-color: $color-gray-500;
   padding: $tilePaddingTop $spacing-md $spacing-md;
 }
 
 .faceBack {
   padding-top: $tilePaddingTop;
   transform: rotateY(180deg);
+  border-color: $color-blue-400;
   background-color: $color-blue-100;
```

Comment:
To investigate the usage of Tiles with a back and let Jess know, as her expectations (without knowing context) would be that the back would either match the front or be the default colour (white).

Reply:
Leaving this for a later design review.